### ISSUE TITLE
Can't build

### ISSUE THREAD

**Issue by daGrevis** [jsdf/coffee-react-transform#1]
First of all, this is exactly what I was looking for and great job!

We want to use React, but we need to be able to write HTML-like “templates“ that we will pass to `render` methods. One solution is Lisp-style trees, but my colleagues don't like that at all so it's out of discussion. Other solution is using backticks to embed JavaScript, but we would much rather use CoffeeScript. This look exactly like something we need.

The problem is that I can't build this — `cake build` returns:

```
File not found: src/csx-transformer.coffee
```

I tried to fix it and eventually got some files in `lib/` directory, but now how could I run them against `.csx` file?

Third thing I would love to see is this package on NPM.

tl;dr
1. You are awesome for trying to solve the same problem we have,
2. How can I build this?
3. How can I run this?
4. You should put this on NPM.
